js/script.js

```diff
@@ -14,8 +14,7 @@ function getQuote() {
 
 function createTweet(input) {
     
-        if(!Array.isArray(input)) {
-            alert('ERROR!');
+        if (!Array.isArray(input) && input.length == 0) {
```

Comment:
`if (!Array.isArray(input) || !input.length) {` - what do you think about that solution? :)

Reply:
It's look better ;)